Statistics/Histograms/Histogram.cs

```diff
@@ -799,6 +824,35 @@ public Int64 EstimateIterationsRemaining(double upperq, double lowerq)
             return Convert.ToInt64(Math.Min(remainingIters, biggestGuess));
         }
 
+        public string Print(bool round = false)
+        {
+            string histogram = $"This histogram consists of the following bin starts and bin counts:" + Environment.NewLine;
+            for (int i = 0; i < BinCounts.Length; i++)
+            {
+                histogram += $"Bin Start: {Min + BinWidth*i}, Bin Count: {BinCounts[i]}" + Environment.NewLine;
+            }
+            return histogram;
+        }
+
+        public string Requirements(bool printNotes)
```

Comment:
This seems strange. I don't think I've seen us do this anywhere else. Do we have a IRequirements or something interface we're adhereing to or is this special for historgram?

Reply:
IDistribution - leftovers from jk

Reply:
we should evaluate this stuff at some point to decide whether it is something we want to keep

Reply:
agreed.